### DISCUSSION TITLE
Is `ToDeviced` transform needed for Cache I/O and GPU transform?

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/tutorials#1404] [Q&A]
Hi!
In the [section "4. Cache IO and transforms data to GPU"](https://github.com/Project-MONAI/tutorials/blob/main/acceleration/fast_model_training_guide.md#4-cache-io-and-transforms-data-to-gpu) in fast training guide, it seems `ToDeviced` transform is needed (after `EnsureTyped`) to move data to GPU. However, in the [fast_training_tutorial.ipynb](https://github.com/Project-MONAI/tutorials/blob/main/acceleration/fast_training_tutorial.ipynb),  `ToDeviced` is not used, and only `EnsureTyped` is used to "convert the data to Tensor without meta, move to GPU and cache to avoid CPU -> GPU sync in every epoch" according to the comment. 

So is `ToDeviced` still needed after `EnsureTyped`? Or just `EnsureTyped` is enough?

**Comment by wyli** [comment]
sorry for the confusion, `ToDeviced` is not needed, `EnsureTyped` should be enough for this use case. I'll update the the descriptions.

**Reply by hw-ju** [reply]
@wyli Thanks a lot for the clarification!

**Reply by hw-ju** [reply]
@wyli Could you also update the description below in the `Define the training progress` section in [fast_training_tutorial.ipynb](https://github.com/Project-MONAI/tutorials/blob/main/acceleration/fast_training_tutorial.ipynb)?
```
ToDeviced transform: to move data to GPU and cache with CacheDataset, then execute random transforms on GPU directly, avoid CPU -> GPU sync in every epoch. Please note that not all the MONAI transforms support GPU operation so far, still working in progress.
```

**Reply by KumoLiu** [reply]
Hi @hw-ju, have updated it in this [PR](https://github.com/Project-MONAI/tutorials/pull/1509). Thanks!
